content/announcing-sigstore-go.md

```diff
@@ -0,0 +1,24 @@
++++
+title = "Announcing sigstore-go"
+author = "The Sigstore Technical Steering Committee"
+date = "2023-10-05"
+draft = false
+type = "post"
+tags = ["sigstore","go","clients"]
++++
+
+
+# Announcing sigstore-go
+
+Today we’re excited to announce a new open source library, [sigstore-go](https://github.com/sigstore/sigstore-go) that represents the future of Sigstore’s support for the Go programming language. Since the beginning, Sigstore has been primarily written in Go but there has been a gap over the past year or so since we established the [Protobufs-based bundle format](https://github.com/sigstore/protobuf-specs): the de facto standard client (Cosign) lacks support for it. sigstore-go represents the TSC’s vision for future integrations in Go and will be used to eventually make Cosign smaller and more performant in the future, as well as provide the basis for Sigstore bundle support in the [policy controller](https://github.com/sigstore/policy-controller).
```

Comment:
You touch on all these points later, so you could omit them from this paragraph, but maybe call out a few other reasons we needed sigstore-go, in addition to lack of the protobuf bundle and dependency size:
* Cosign as a library is designed primarily to support OCI structures, so it's difficult to use for library integrators that only need to sign/verify binaries
* Lots of duplication throughout Cosign due to multiple verification options
* sigstore/sigstore has been a catch-all library to provide core utilities across Go-based Sigstore projects, but it's confusingly named